scripts/t265_to_mavlink.py

```diff
@@ -578,13 +578,17 @@ def user_input_monitor():
                 # Check for pose jump and increment reset_counter
                 if prev_data != None:
                     delta_translation = [data.translation.x - prev_data.translation.x, data.translation.y - prev_data.translation.y, data.translation.z - prev_data.translation.z]
+                    delta_velocity = [data.velocity.x - prev_data.velocity.x, data.velocity.y - prev_data.velocity.y, data.velocity.z - prev_data.velocity.z]
                     position_displacement = np.linalg.norm(delta_translation)
+                    speed_delta = np.linalg.norm(delta_velocity)
 
                     # Pose jump is indicated when position changes abruptly. The behavior is not well documented yet (as of librealsense 2.34.0)
                     jump_threshold = 0.1 # in meters, from trials and errors, should be relative to how frequent is the position data obtained (200Hz for the T265)
-                    if (position_displacement > jump_threshold):
-                        send_msg_to_gcs('Pose jump detected')
+                    jump_speed_threshold = 20.0 # in m/s from trials and errors, should be relative to how frequent is the velocity data obtained (200Hz for the T265)
+                    if (position_displacement > jump_threshold) and (speed_delta > jump_speed_threshold):
```

Comment:
Hi @apinxiko , thank you very much for your contribution. So sorry for the getting to it this late.

From my tests, the "pose jump" happens when I can see a visible jump, but I did not see any big changes in the velocity data itself, so probably the logic here should be "or" instead of "and". 

If you have experimented and observe that it is indeed the case (both translation and velocity jumps at the same time), then it still works.